doc/user/content/metrics.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Metrics"
```

Comment:
Should go under "Operating Materialize"?

Reply:
Same for `System Catalog`.

Reply:
deleted metrics.

Reply:
I see the division as:

![image](https://user-images.githubusercontent.com/11527560/163581609-2da06de2-ce13-4333-98b2-dc1199d288a0.png)

Operating Materialize is all "How-To" and System catalog is more reference